### DISCUSSION TITLE
Mac Safari 浏览器沉浸式翻译插件提示版本过旧

### DISCUSSION THREAD

**Discussion by HahaCodeLife** [immersive-translate/immersive-translate#1898] [Help] [answered]
### 概述 | Summary

请及时更新你们的应用商店的新版本提交

**Comment by keyboard3** [comment]
截个图看下，大概率是低版本的macOS

**Reply by HahaCodeLife** [reply]
Big Sur 这个还分 MacOS 版本吗？😄
<img width="698" alt="image" src="https://github.com/user-attachments/assets/f288b2ec-4389-4f12-9229-d0fd3cc26304">

老版本的有办法更新吗？

**Reply by keyboard3** [reply, marked as answer]
太老了，至少12版本以上。 建议用chrome浏览器

**Reply by HahaCodeLife** [reply]
好的
